README.md

```diff
@@ -79,15 +87,45 @@ The Cards and Markers can also leverage SwiftUI Viewbuilders allowing for custom
 <img height="400" alt="rcDemo1" src="https://user-images.githubusercontent.com/77754056/119742939-784d7200-be4e-11eb-928d-6d83b07e49ff.png">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img height="400" alt="rcDemo2" src="https://user-images.githubusercontent.com/77754056/119743047-a6cb4d00-be4e-11eb-8543-4ed018fa25f3.jpeg">
 </p>
 
-##### Data Consumption
+## Data Consumption
+
+### CardItemModel
+
+The data thats associated with the Annotations from the scene are loaded into the project as a list of app developer `CardItem` models that must conform to `CardItemComponent`. The *name* of the Entity (sphere) from Reality Composer corresponds to the *id* property of the Model. The list of initial CardItems are passed into the chosen loading strategy with the scene.
+
+### JSON
+
+Each of the loading strategies also have an initializer for accepting the data from a top level JSON array. The data that's provided by JSON will internally be constrained to using `DecodableCardItem`, a provided concrete implementation of the `CardItemModel`.
+
+```swift
+// JSON key/value:
+"id": String,
+"title_": String,
+"descriptionText_": String?,
+"detailImage_": Data?, // base64 encoding of Image
+"actionText_": String?,
+"icon_": String? // systemName of SFSymbol
+```
+
+### Loading Strategies
+
+The supported loading strategies (`UsdzFileStrategy`, `RealityFileStrategy`, and `RCProjectStrategy`) consume the initial data required to create the AR Experience and bind the locations from the Reality Composer scene to the data for each `MarkerView` and `CardView`. For an `Image` anchor, the image that will be detected must be passed into the strategies with the physical width of the real world image to create an `ARReferenceImage` for detection. For an `Object` anchor the anchorImage and physicalWidth can be nil.
 
-CardItem Models Conform to `CardItemComponent`. The *name* of the Entity (Sphere) from Reality Composer corresponds to the *id* property of the Model. The list of initial CardItems are passed into the `RealityComposerStrategy` with the Reality Composer rcproject File Name and the name of the scene. For an image Anchor, the image that will be detected must be passed into the strategy to create an `ARReferenceImage`.
+The scene can be represented in different file types and each strategy requires different data and setup.
+- **USDZ Strategy:** Requires a URL path to the `.usdz` file
+- **Reality Strategy:** Requires a URL path to the `.reality` file and the name of the scene 
+- **RCProject Strategy:** The saved `.rcproject` can be dragged into XCode and the name of the `.rcproject` file and the scene name are required.
 
-##### Creating the ContentView and loading the data
+> **Note**:
+- RCProject strategy requires the .rcproject file as a resource in XCode at build time
```

Comment:
```suggestion
- The RCProject strategy requires that the `.rcproject` file is part of the application bundle so that the file is available already during build time. Drag the file into Xcode to do so.
```